android/app/build.gradle

```diff
@@ -92,6 +92,14 @@ android {
             keyAlias 'androiddebugkey'
             keyPassword 'android'
         }
+        release {
+            storeFile file('release.keystore')
+            // Replace with keystore password
+            storePassword '<release-keystore-password>'
```

Comment:
we should later move this to env variables so we don't need to manually replace them every time